libmscore/edit.cpp

```diff
@@ -3075,4 +3075,176 @@ void Score::checkSpanner(int startTick, int endTick)
+                        Beam* nb = ocr->beam()->clone();
+                        nb->clear();
+                        nb->setTrack(dtrack);
+                        nb->setScore(score);
+                        nb->add(ncr);
+                        ncr->setBeam(nb);
+                        }
+
+                  Tuplet* ot = ocr->tuplet();
+
+                  // clone Tuplets
+                  if (ot) {
+                        ot->setTrack(strack);
+                        Tuplet* nt = tupletMap.findNew(ot);
+                        if (nt == 0) {
+                              if (link)
+                                    nt = static_cast<Tuplet*>(ot->linkedClone());
+                              else
+                                    nt = static_cast<Tuplet*>(ot->clone());
+                              nt->setTrack(dtrack);
+                              nt->setParent(dm);
+                              tupletMap.add(ot, nt);
+
+                              Tuplet* nt1 = nt;
+                              while (ot->tuplet()) {
+                                    Tuplet* nt = tupletMap.findNew(ot->tuplet());
+                                    if (nt == 0) {
+                                          if (link)
+                                                nt = static_cast<Tuplet*>(ot->tuplet()->linkedClone());
+                                          else
+                                                nt = static_cast<Tuplet*>(ot->tuplet()->clone());
+                                          nt->setTrack(dtrack);
+                                          nt->setParent(dm);
+                                          tupletMap.add(ot->tuplet(), nt);
+                                          }
+                                    nt->add(nt1);
+                                    nt1->setTuplet(nt);
+                                    ot = ot->tuplet();
+                                    nt1 = nt;
+                                    }
+                              }
+                        nt->add(ncr);
+                        ncr->setTuplet(nt);
+                        }
+
+                  if (oe->isChordRest()) {
+                        if (oe->isChord()) {
+                              Chord* och = static_cast<Chord*>(ocr);
+                              Chord* nch = static_cast<Chord*>(ncr);
+
+                              int n = och->notes().size();
+                              for (int i = 0; i < n; ++i) {
+                                    Note* on = och->notes().at(i);
+                                    Note* nn = nch->notes().at(i);
+                                    if (on->tieFor()) {
+                                          Tie* tie;
+                                          if (link)
+                                                tie = static_cast<Tie*>(on->tieFor()->linkedClone());
```

Comment:
Things in edit.cpp are wrong in my other PR, too then but if we assume this one is merged after the other...